### PR TITLE
Add 'Add People' button on chat info panel

### PR DESCRIPTION
**Pull request by akalin-keybase**
Have it replace '+ Add people' that was all the way at the bottom.

Add currently-disabled item for 'To this channel', which will be done in another PR.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
screenshot from 2019-02-12 18-26-33

**Comment by akalin-keybase** [comment]
Oops, forgot to tag @keybase/react-hackers

File: shared/chat/conversation/info-panel/add-people.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+type Props = {
+  attachTo: () => ?React.Component<any>,
+  visible: boolean,
+  onAddPeople: () => void,
+  onHidden: () => void,
+}
+
+const AddPeopleHow = (props: Props) => {
+  const items = [{onClick: props.onAddPeople, title: 'To team'}, {disabled: true, title: 'To channel'}]
```
Comment (buoyad) [inline shared/chat/conversation/info-panel/add-people.js]: No 'To channel' option for small teams, also the spacing seems off in the small team info pane:

image

**Comment by akalin-keybase** [comment]
Closing this, as my original plan was to implement the 'To channel' immediately after this. Since we're not doing that, this should probably go in only when someone picks that up.

**Comment by buoyad** [comment]
@keybase/react-hackers this is ready again

File: shared/chat/conversation/info-panel/add-people.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+)(AddPeopleHow)
+
+const _AddPeople = (props: {smallTeam: boolean, teamname: string} & OverlayParentProps) => {
+  return (
+    <Box2 direction="horizontal" centerChildren={true}>
+      <ConnectedAddPeopleHow
+        attachTo={props.getAttachmentRef}
+        visible={props.showingMenu}
+        teamname={props.teamname}
+        showChannelOption={!props.smallTeam}
```
Comment (akalin-keybase) [inline shared/chat/conversation/info-panel/add-people.js]: Without the show channel option, the menu only has one item. Maybe the button should just go to the add-to-team dialog directly?

Comment (buoyad) [reply shared/chat/conversation/info-panel/add-people.js]: Good idea, this is also true for `#general` in big teams. I changed the semantics a bit to `isGeneralChannel`

**Pull request merged**